src/ur/workday.urs

```diff
@@ -0,0 +1,145 @@
+
+type descriptor = {
+     Id : wid,
+     Descriptor : option string
+}
+
+type comment = {
+     Date : time,
+     Comment : string,
+     Person : descriptor (* who made the comment *)
+}
+type attachment = {
+     Id : wid,
+     Description : string,
+     ContentType : descriptor,
+     FileLength : int,
+     UploadDate : time,
+     FileName : string,
+     UploadedBy : descriptor,
+     Category : descriptor
+}
+
+type businessProcessParameters = {
+     For : option descriptor, (* could be another business process if this is a subprocess *)
+     TransactionStatus : option descriptor,
+     Action : option descriptor, (* required *)
+     OverallBusinessProcess : option descriptor,
+     Comment : option string,
+     Comments : option (list comment),
+     WarningValidations : option string,
+     Attachments : option (list attachment),
+     OverallStatus : option string,
+     CriticalValidations : option string,
+}
+type feedbackResponse = {
+     Id : wid,
+     Response : string,
+     Declined : bool,
+     DeclineReason : string,
+     Responder : descriptor,
+     Preview : string
+}
+
+type feedbackQuestion = {
+     Id : wid,
+     Question : string,
+     Responses : list feedbackResponse,
+     QuestionType : descriptor,
+     RelatesTo : list descriptor,
+     MultipleChoiceAnswers : list descriptor,
+     Preview : string
+}
+
+type relatedFeedback = {
+     Id : wid,
+     BusinessProcessParameters : businessProcessParameters,
+     Preview : string
+}
+
+type anytimeFeedback = {
+     Id : wid,
+     Preview : option string,
+     ToWorker : option descriptor, (* who the feedback is about *)
+     WorkersToNotify : option (list descriptor), (* always the direct manager *)
+     BusinessProcessParameters : option businessProcessParameters,
+     HiddenFromWorker : option bool, (* True if the feedback event is confidential between the feedback provider and the manager. Workers don't see confidential feedback. *)
+     HiddenFromManager : option bool, (* True if the feedback event is private between the feedback provider and the worker. Private feedback isn't shared with managers. *)
+     Comment : option string, (* actual feedback *)
+     FromWorker : option descriptor, (* Identity of the responder. Only provided if the request asked to know the identity of the responder. *)
+     OverallStatus : option string,
+     GivenDate : option time,
+     ShowFeedbackProviderName: option bool,
+     (* unused parameters: *)
+     (*
+     Badge : descriptor,
+     Href : string
+     AlsoAbout : list descriptor, (* other workers the feedback is about *)
+     RelatedFeedbackEvents : list relatedFeedback,
+     RelatesTo : descriptor,
+      *)
+}
+val json_anytimeFeedback : Json.json anytimeFeedback
+
+datatype feedbackStatus = Complete | Requested
+(*
+val show_feedbackStatus : show feedbackStatus
+val read_feedbackStatus : read feedbackStatus
+val eq_feedbackStatus : eq feedbackStatus
+*)
```

Comment:
Commenting nit: I think the `datatype` should be commented out too, since it's not being used.  Also, the note about how a `sql_injectable` instance is not possible makes sense here to describe why this exists but is commented out.